.circleci/config.yml

```diff
@@ -6,52 +6,63 @@ jobs:
-            sleep 10
-            oc login -u system:admin
-            echo "Configured OpenShift cluster : v3.9.0"
-
-      # based on this: https://discuss.circleci.com/t/circle-2-0-caching-is-too-limited-to-be-very-useful/11694/10
-      - restore_cache:
-          key: spring-boot-http-secured-booster-{{ .Branch }}-{{ checksum "pom.xml" }}
-          key: spring-boot-http-secured-booster-{{ .Branch }}
-          key: spring-boot-http-secured-booster
-
-      # Will download all deps needed into .m2
-      - run: ./mvnw dependency:go-offline
-
-
-      - save_cache:
-          paths:
-            - ~/.m2
-          key: spring-boot-http-secured-booster-{{ .Branch }}-{{ checksum "pom.xml" }}
+#      - run:
+#          command: |
+#            kube_version=$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)
+#            curl -LO https://storage.googleapis.com/kubernetes-release/release/${kube_version}/bin/linux/amd64/kubectl && \
+#                chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+#            echo "Installed kubectl CLI tool"
+#
+#            oc_tool_version="openshift-origin-server-v3.9.0-191fece-linux-64bit"
+#            curl -LO https://github.com/openshift/origin/releases/download/v3.9.0/${oc_tool_version}.tar.gz && \
+#                tar -xvzf ${oc_tool_version}.tar.gz && chmod +x $PWD/${oc_tool_version}/oc && sudo mv $PWD/${oc_tool_version}/oc /usr/local/bin/ && \
+#                rm -rf ${oc_tool_version}.tar.gz
+#            echo "Installed OC CLI tool"
+#
+#            tmp=`mktemp`
+#            echo 'DOCKER_OPTS="$DOCKER_OPTS --insecure-registry 172.30.0.0/16"' > ${tmp}
+#            sudo mv ${tmp} /etc/default/docker
+#            sudo mount --make-shared /
+#            sudo service docker restart
+#            echo "Configured Docker daemon with insecure-registry"
+#
+#            oc cluster up --version=v3.9.0
+#            sleep 10
+#            oc login -u system:admin
+#            echo "Configured OpenShift cluster : v3.9.0"
+#
+#      # based on this: https://discuss.circleci.com/t/circle-2-0-caching-is-too-limited-to-be-very-useful/11694/10
+#      - restore_cache:
+#          key: spring-boot-health-booster-{{ .Branch }}-{{ checksum "pom.xml" }}
+#          key: spring-boot-health-booster-{{ .Branch }}
+#          key: spring-boot-health-booster
+#
+#      # Will download all deps needed into .m2
+#      - run: ./mvnw dependency:go-offline
+#
+#
+#      - save_cache:
+#          paths:
+#            - ~/.m2
+#          key: spring-boot-health-booster-{{ .Branch }}-{{ checksum "pom.xml" }}
+#
+#      - run:
+#          command:  ./mvnw clean verify -Popenshift,openshift-it
+#          no_output_timeout: 3600
 
       - run:
+          name: echo vars
           command: |
-            oc apply -f service.sso.yaml
-            ./mvnw clean verify -Popenshift,openshift-it -DSSO_AUTH_SERVER_URL=$(oc get route secure-sso -o jsonpath='https://{.spec.host}/auth')
```

Comment:
How is the SSO server started now?

Reply:
It's not :).

I started working on this by mistake. I wanted to work on health check, but opened the wrong project :P

Reply:
So this is still work in progress

Reply:
Actually, we don't have an s2i build for this booster.

Reply:
That would explain why I didn't find specific configuration  in our Jenkins Groovy scripts.

Reply:
Any reason why we don't have s2i tests for this booster?

Reply:
I don't recall the specifics but that decision was made at the RHOAR level. I think it had to do with the complexity of the scenario but I'm not sure. @cmoulliard might recall.

Reply:
Thanks! @Ladicek any idea why we don't include s2i tests for the http-secured booster?

Reply:
The Secured mission doesn't have an S2I variant. I know some boosters do implement it (Vert.x and SB), but some don't (I'm sure WildFly Swarm doesn't, not sure about Node.js). It's .zip-only in the catalog. There's a JIRA (OBST-440), but it looks dormant. We'd need someone to start pushing for it :-) On the testing front, it would be pretty easy.

Reply:
Thanks for the update @Ladicek!